custom_components/tahoma/sensor.py

```diff
@@ -87,7 +84,7 @@
     "core:TemperatureInCelsius": TEMP_CELSIUS,
```

Comment:
This will also make it harder by the way. If we move all sensor entities to this new structure, we cannot use this list easily. `"core:MeasuredValueType"` does only exist once for a device, where it can have multiple states. We can only use this when we know what the primary state is of a certain ui class/widget.